apptools/apptools-windows-tests/apptools/update.py

```diff
@@ -41,36 +41,15 @@ class TestCrosswalkApptoolsFunctions(unittest.TestCase):
     def test_update_no_argument(self):
         comm.setUp()
         comm.create(self)
-        updatecmd = comm.HOST_PREFIX + comm.PackTools + "crosswalk-app update org.xwalk.test --windows-crosswalk=" + comm.XwalkPath + comm.windowsCrosswalk
-        (return_update_code, update_output) = comm.getstatusoutput(updatecmd)
-        comm.clear("org.xwalk.test")
-        self.assertIn("ERROR:", update_output[0])
-
-    def test_update_beta(self):
-        comm.setUp()
-        comm.create(self)
-        updatecmd = comm.HOST_PREFIX + comm.PackTools + "crosswalk-app update beta org.xwalk.test --windows-crosswalk=" + comm.XwalkPath + comm.windowsCrosswalk
-        comm.update(self, updatecmd)
-        comm.clear("org.xwalk.test")
-
-    def test_update_canary(self):
-        comm.setUp()
-        comm.create(self)
-        updatecmd = comm.HOST_PREFIX + comm.PackTools + "crosswalk-app update canary org.xwalk.test --windows-crosswalk=" + comm.XwalkPath + comm.windowsCrosswalk
+        os.chdir('org.xwalk.test')
+        updatecmd = comm.HOST_PREFIX + comm.PackTools + "crosswalk-app update --windows-crosswalk=" + comm.XwalkPath + comm.windowsCrosswalk
```

Comment:
"update" command will be depreciated in windows platform, please remove related tests.